src/app.ts

```diff
@@ -73,7 +73,6 @@ async function app() {
   }
 
   await Promise.all(dependencyDefs);
-  fhirdefs.loadFromPath(inDir, '', defs);
```

Comment:
Note that we don't need to load the `inDir` into the `defs` anymore because we now manage if in the `LakeOfFHIR`.